mobile_app_connector/models/app_hub.py

```diff
@@ -98,29 +101,48 @@ def mobile_get_message(self, partner_id, **pagination):
             "correspondence": letters,
         }
         unpaid_data = {"recurring.contract": unpaid}
-        # TODO handle pagination properly
-        limit = int(pagination.get("limit", 1000))
-        _logger.info("BEGIN RENDER TILES")
-        messages = available_tiles[:limit].render_tile(tile_data)
-        _logger.info("END RENDER TILES")
-
-        # GI7 is treated separately because it needs unpaid sponsorships
-        msg_tmp = (
-            self.env["mobile.app.tile"]
-            .search(
-                [
-                    (
-                        "subtype_id",
-                        "=",
-                        self.env.ref("mobile_app_connector.tile_subtype_gi7").id,
-                    ),
-                    ("create_date", "!=", False),
-                ]
+
+        app_messages = partner.app_messages
+
+        start = int(pagination.get("start", 0))
+        limit = int(pagination.get("limit", 10))
+        if start != 0:
```

Comment:
Maybe we should not have automatic creation of app.messages for every partner and just check here if messages are available:
```suggestion
        if start != 0 and app_messages:
```

Reply:
Also, you don't check for last refresh date? Maybe if the date is too old it would be good to still load the hub.

Reply:
Yes, if we don't add to every partner an app.message entry, then this is required. We could potentially use this to create an entry for partner that does not have one yet.
For the refresh date, this can be easily implemented, but is there nothing in the app for a refresh of the hub after a certain amount of time ?